lib/modules/dosomething/dosomething_mbp/dosomething_mbp.install

```diff
@@ -4,14 +4,44 @@
  * Installation and schema hooks for dosomething_mbp.module.
  */
  
- /**
+/**
+ * Implements hook_install().
+ */
+function dosomething_mbp_install() {
+
+   // Default Message Broker RabbitMQ connection settings
+   variable_set('message_broker_producer_rabbitmq_host', getenv('DS_MB_RABBITMQ_HOST'));
```

Comment:
@sergii-tkachenko Do the environment variables get set in `/etc/environment` on our Vagrant box? Is this the same for the production server as well?